### PR TITLE
Future days constraint

### PR DESCRIPTION
**Pull request by folix-01**

### CODE REVIEW

**Comment by folix-01** [comment]
The broking test is from master, and it also brokes on master. Idk why

File: src/redturtle/prenotazioni/locales/redturtle.prenotazioni.pot [hunk 1]
```diff
@@ -477,6 +477,10 @@ msgstr ""
 msgid "Unknown booking type '${booking_type}'."
 msgstr ""
 
+#: redturtle/prenotazioni/content/prenotazioni_folder.py:149
+msgid "Usage of `future days` field requires the `fiscalcode` to be between the required fields."
```
Comment (mamico) [inline src/redturtle/prenotazioni/locales/redturtle.prenotazioni.pot:481]: serve la traduzione in itialiano

Comment (folix-01) [reply src/redturtle/prenotazioni/locales/redturtle.prenotazioni.pot:481]: SI, è nel file pot italiano

**Comment by folix-01** [comment]
Changed the target field for validator, max bookings num checked now

**Pull request merged**